backup.go

```diff
@@ -175,7 +183,8 @@ func (l *KVLoader) send() error {
 		return err
 	}
 
-	l.entries = make([]*Entry, 0, 1000)
+	l.entries = l.entries[:0]
```

Comment:
We should not do this as this will modify entries slice already passed to db.batchSetAsync.